dtls/client-dtls-resume.c

```diff
@@ -118,22 +95,56 @@ int main (int argc, char** argv)
 
     wolfSSL_set_fd(ssl, sockfd);
     if (wolfSSL_connect(ssl) != SSL_SUCCESS) {
-	    int err1 = wolfSSL_get_error(ssl, 0);
-	    char buffer[80];
-	    printf("err = %d, %s\n", err1, wolfSSL_ERR_error_string(err1, buffer));
-	    printf("SSL_connect failed");
+        int err1 = wolfSSL_get_error(ssl, 0);
+        char buffer[80];
+        printf("err = %d, %s\n", err1, wolfSSL_ERR_error_string(err1, buffer));
+        printf("SSL_connect failed");
         return 1;
     }
 
-    DatagramClient(ssl);
+/*****************************************************************************/
+/*                     Code for sending datagram to server                   */
+    int     recvlen;
```

Comment:
Please more variable declarations to top of the function, thanks.  Apply this comment to all files.